api/task.go

```diff
@@ -239,6 +239,7 @@ type TaskCreate struct {
 	Payload           string
 	EarliestAllowedTs int64  `jsonapi:"attr,earliestAllowedTs"`
 	Statement         string `jsonapi:"attr,statement"`
+	SheetID           int    `jsonapi:"attr,sheetId"`
```

Comment:
I don't think we should support updating issue tasks if they use sheet. If they need to update statement, they can update the sheet content. If a task failed in the middle, they can create a new issue for failed databases to complete the rest of the content. Thoughts @Candybase?

Reply:
Done